cql2pgjson/src/test/resources/templates/db_scripts/courseReservesSchema.json

```diff
@@ -0,0 +1,187 @@
+{
```

Comment:
The test should be as concise as possible. Adding the complete courseReservesSchema.json is not needed and makes it difficult to understand the key point of the test.

For foreign key tests we already have https://github.com/folio-org/raml-module-builder/blob/master/cql2pgjson/src/test/resources/templates/db_scripts/foreignKey.json that can be extended by an entry in a"foreignKeys" list where "fieldName" contains a dot because it is a sub-field.

Reply:
done

Reply:
courseReservesSchema.json should be removed if it is no longer in use.